lib/apipie/validator.rb

```diff
@@ -374,6 +374,27 @@ def description
       end
     end
 
+    class DecimalValidator < BaseValidator
+
+      def validate(value)
+        self.class.validate(value)
+      end
+
+      def self.build(param_description, argument, options, block)
+        if argument == :decimal
+          self.new(param_description)
+        end
+      end
+
+      def description
+        "Must be a decimal number."
+      end
+
+      def self.validate(value)
+        value.to_s =~ /\A[0-9]+([,.][0-9]+)?\Z$/
```

Comment:
What do you all think of validating for negative decimals as well? This would keep this validator more in line with the `IntegerValidator`, which also allows negatives:
```ruby
value.to_s =~ /\A^[-+]?[0-9]+([,.][0-9]+)?\Z$/
```

Reply:
Yeah, good call